docs/self-managed/platform-deployment/helm-kubernetes/platforms/redhat-openshift.md

```diff
@@ -172,7 +172,7 @@ helm install <RELEASE_NAME> camunda/camunda-platform --skip-crds -f values.yaml
 If you must deploy using Helm 3.2.0 or greater, you have two options. One is to use a SCCs which defines the `RunAsUser` strategy to be at least `RunAsAny`. If that's not possible, then you need to make use of [a post-renderer](https://helm.sh/docs/topics/advanced/#post-rendering). This workaround is also described in detail in the [Helm chart repository](https://github.com/camunda/camunda-platform-helm/tree/main/openshift#using-a-post-renderer).
 
 :::warning
-If using a post-renderer, you **must** use the post-renderer whenever you are upgrading your release, not only during the initial installation. If you do not, the default values will be used again, which will prevent some services from starting.
+If using a post-renderer, you **must** use the post-renderer whenever you are updating your release, not only during the initial installation. If you do not, the default values will be used again, which will prevent some services from starting.
```

Comment:
Similar concern to Helm, **upgrade** was the terminology used by Red Hat Openshift. Interesting here, though, it looks like they've shifted to **update**!

* https://docs.openshift.com/container-platform/3.11/upgrading/index.html
> When new versions of OpenShift Container Platform are released, you can upgrade your existing cluster to apply the latest enhancements and bug fixes.
* https://docs.openshift.com/container-platform/4.12/updating/index.html
> You can update an OpenShift Container Platform 4 cluster with a single operation by using the web console or the OpenShift CLI (oc).

Reply:
Sounds like several groups are having the same conversation we are 😄